src/applause-button.js

```diff
@@ -40,26 +34,18 @@ const toggleClass = (element, cls) => {
 };
 
 const debounce = (fn, delay) => {
-  var timer = null;
-  return function() {
-    var context = this,
+  let timer = null;
+  return function () {
+    const context = this,
       args = arguments;
     clearTimeout(timer);
     timer = setTimeout(() => fn.apply(context, args), delay);
   };
 };
 
-// https://github.com/WebReflection/document-register-element#v1-caveat
```

Comment:
is this baked into the ungap import?

Reply:
Yes. Now that pretty much all browsers support Custom Elements that _extend_ native elements, we no longer need this old-style workaround with its constructor hack.

See the repo for info:
https://github.com/WebReflection/document-register-element?tab=readme-ov-file#deprecated